### PR TITLE
Add SyncMaxTS method

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

Part of tikv/pd#2759.

### CODE REVIEW

**Review by Yisaer — COMMENTED** [review]
Basically LGTM. I'm wondering maybe we also need to add the sender information like memberID in request. And each receiver shoudl compare the sender member ID and the leaderid in their vision.

**Review by Yisaer — APPROVED** [review]
LGTM @disksing  PTAL

**Review by disksing — APPROVED** [review]

File: proto/pdpb.proto [hunk 1]
```diff
@@ -627,4 +627,5 @@ message SyncMaxTSRequest {
 message SyncMaxTSResponse {
     ResponseHeader header = 1;
     Timestamp max_local_ts = 2;
+    repeated string synced_dcs = 3;
```
Comment (disksing) [inline proto/pdpb.proto:630]: Maybe just `dcs`, for the first phase they do not sync anything.

Comment (JmPotato) [reply proto/pdpb.proto:630]: nice idea.

**Review by disksing — APPROVED** [review]

**Pull request merged**